### PR TITLE
hyperkitty: 1.3.3 -> 1.3.4

### PR DESCRIPTION
**Pull request by alyssais**
Tested on my deployment.  Seems to work.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/flufl/lock.nix [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+  version = "5.0.5";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0nzzd6l30ff6cwsrlrb94xzfja4wkyrqv3ydc6cz0hdbr766mmm8";
+    sha256 = "1bnapkg99r6mixn3kh314bqcfk8q54y0cvhjpj87j7dhjpsakfpz";
+  };
+
+  propagatedBuildInputs = [ atpublic psutil ];
+  checkInputs = [ pytest ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/flufl/lock.nix:13]:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

Comment (alyssais) [reply pkgs/development/python-modules/flufl/lock.nix:13]: The manual says

> pytestCheckHook is a convenient hook which will substitute the setuptools test command for a checkPhase which runs pytest. This is also beneficial when a package may need many items disabled to run the test suite.

Why should I use it here?

Comment (SuperSandro2000) [reply pkgs/development/python-modules/flufl/lock.nix:13]: Because setuptools test will no longer be called in some future.

Comment (alyssais) [reply pkgs/development/python-modules/flufl/lock.nix:13]: Thanks for explaining.  Could we get that in the manual maybe?

**Pull request merged**